### PR TITLE
Allow replicatedMap bean creation in spring context

### LINKED ISSUE #4958: <hz:replicatedMap> cannot be created outside of <hz:config> element
The schema doesn't allow for an explict <hz:replicatedMap> element to be created.  One can be created inside <hz:config> but not as a definition for a concrete replicated Map.

Therefore at present it is impossible to define a replicatedMap using Spring.

### PR DESCRIPTION
**Pull request by emrahkocaman**
Fixes #4958

### CODE REVIEW

File: hazelcast-spring/src/main/resources/hazelcast-spring-3.5.xsd [hunk 1]
```diff
@@ -1017,6 +1017,18 @@
             </xs:appinfo>
         </xs:annotation>
     </xs:element>
+    <xs:element name="replicatedMap" type="hazelcast-type">
+        <xs:annotation>
+            <xs:documentation>
+                Retrive an hazelcast ReplicatedMap instance
```
Comment (mesutcelik) [inline hazelcast-spring/src/main/resources/hazelcast-spring-3.5.xsd]: retrive --> retrieve 
seems it is all over the file.

**Comment by mesutcelik** [comment]
:+1: 
minor comment. 

Can you please create backport? 
Do we need to update code samples?

File: hazelcast-spring/src/main/resources/hazelcast-spring-3.5.xsd (hunk 1 above)
Comment (Donnerbart) [inline hazelcast-spring/src/main/resources/hazelcast-spring-3.5.xsd]: an -> a
hazelcast -> Hazelcast

Comment (mesutcelik) [inline hazelcast-spring/src/main/resources/hazelcast-spring-3.5.xsd]: http://editingandwritingservices.com/a-or-an-before-words-beginning-with-h/

I think @Serdaro can help us on that. :)

**Comment by emrahkocaman** [comment]
Typos fixed, backport created, code-samples updated. (https://github.com/hazelcast/hazelcast-code-samples/commit/72299832a62b25f628e5ee99a4a9794556be794f)

**Comment by mesutcelik** [comment]
verify

**Pull request merged**